bigquery/bigquery.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: cloud-build-notifiers/v1alpha1
+kind: BigQueryNotifier
+metadata:
+  name: example-bigquery-notifier
+spec:
+  notification:
+    filter: build.status == Build.Status.SUCCESS
```

Comment:
Fix: This notifier and the data it stores will be most useful when used for triggers, not statuses. I recommend the following:

`filter: build.build_trigger_id == "123e4567-e89b-12d3-a456-426614174000"`

Reply:
Resolved!